lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php

```diff
@@ -83,7 +83,7 @@ public function getDatabase(Connection $conn) : ?string
     {
         $params = $conn->getParams();
 
-        return $params['path'] ?? null;
+        return $params['path'] ?? '';
```

Comment:
The returned NULL implies that there's no database currently selected, however, SQLite doesn't support databases and one is always selected. Some of the platform methods do not allow NULL for `$database` which previously was only documented but not enforced.